src/segment-loader.js

```diff
@@ -754,6 +755,7 @@ export default class SegmentLoader extends videojs.EventTarget {
     if (request === this.xhr_.segmentXhr) {
       // the segment request is no longer outstanding
       this.xhr_.segmentXhr = null;
+      segmentInfo.startOfAppend = Date.now();
```

Comment:
in a fragmented mp4 and/or encrypted stream case, we have to wait for all our requests (init segment and/or key), depending on the order we receive the responses, setting the date here will affect the throughput calculation. Not sure if this behavior is ok or not